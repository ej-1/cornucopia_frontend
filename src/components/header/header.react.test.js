import React from "react";
import Header from "../header/header";
import renderer from "react-test-renderer";

test("Header gets rendered", () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
