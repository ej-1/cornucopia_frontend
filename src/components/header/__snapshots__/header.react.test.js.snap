// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header gets rendered 1`] = `
<div
  className="header"
>
  <nav
    className="navbar navbar-inverse"
  >
    <div
      className="container"
    >
      <div
        className="navbar-header"
      >
        <a
          className="navbar-brand"
          href="/"
        >
          CornucopiaTrader
        </a>
        <button
          className="navbar-toggle collapsed"
          onClick={[Function]}
          type="button"
        >
          <span
            className="sr-only"
          >
            Toggle navigation
          </span>
          <span
            className="icon-bar"
          />
          <span
            className="icon-bar"
          />
          <span
            className="icon-bar"
          />
        </button>
      </div>
      <div
        aria-expanded={null}
        className="navbar-collapse collapse"
      >
        <ul
          className="nav navbar-nav"
          role={null}
        >
          <li
            className=""
            role="presentation"
            style={undefined}
          >
            <a
              href="/simulate"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              Trading Strategy Simulator
            </a>
          </li>
          <li
            className=""
            role="presentation"
            style={undefined}
          >
            <a
              href="/robotconfig"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              Robot Settings
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
</div>
`;
